code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -11,11 +12,25 @@ GLOBAL_LIST_INIT(hardcoded_gases, list(/datum/gas/oxygen, /datum/gas/nitrogen, /
 		gas_info[META_GAS_MOLES_VISIBLE] = initial(gas.moles_visible)
 		if(initial(gas.moles_visible) != null)
 			gas_info[META_GAS_OVERLAY] = new /obj/effect/overlay/gas(initial(gas.gas_overlay))
-		gas_info[META_GAS_FUSION_POWER] = initial(gas.dangerous)
+		gas_info[META_GAS_FUSION_POWER] = initial(gas.fusion_power)
 		gas_info[META_GAS_DANGER] = initial(gas.dangerous)
 		gas_info[META_GAS_ID] = initial(gas.id)
 		.[gas_path] = gas_info
 
+/proc/nonreactive_gas_list() //all the gases not included in _any_ reaction
+	var/list/nonreactive_gases = GLOB.meta_gas_info.Copy()
+	for(var/g in GLOB.meta_gas_info)
+		var/datum/gas/gas = g
+		for(var/r in SSair.gas_reactions)
+			if(!gas in nonreactive_gases) //if its been removed already we can move on
```

Comment:
Pretty sure the precedence is bad here, coming out to `(!gas) in nonreactive_gases`, but also this proc is never called.

Reply:
Would prefer to fix or delete it just so if someone does come to use the proc in the future they're not caught off-guard.

Reply:
i'll just delete the proc, it is kind of pointless anyway